lib/di/module/network_module.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter_survey/api/api_service.dart';
+import 'package:flutter_survey/di/provider/dio_provider.dart';
+import 'package:flutter_survey/env.dart';
+import 'package:injectable/injectable.dart';
+
+@module
+abstract class NetworkModule {
+  @Singleton(as: ApiService)
+  ApiServiceImpl provideApiService(DioProvider dioProvider) {
+    return ApiServiceImpl(
+      dioProvider.getDio(),
+      baseUrl: Env.restApiEndpoint,
```

Comment:
Why don't we keep an [`.env.sample`](https://github.com/nimblehq/flutter-templates/blob/develop/template/.env.sample) file to clarify how many credentials we should define in the app? The external devs can easily clone the sample file and put valid credentials to run the project.